parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -209,6 +223,39 @@ ensureAPMZ() {
     ln -Ffs "$bin_path" "$apmz_filepath" # symlink apmz into /usr/local/bin/apmz
 }
 
+# install mostly-static bpftrace binary linked only against glibc > 2.23
+# it should be comptaible with any linux distro using a new enough kernel.
+# this may in the future require filtering the host OS if we encounter compatibility issues.
+# we also install the most commonly used bpftrace scripts by default.
+installBpftrace() {
+    local version=$1
+    local bpftrace_bin="bpftrace"
+    local bpftrace_tools="bpftrace-tools.tar"
+    local bpftrace_url="https://upstreamartifacts.blob.core.windows.net/$bpftrace_bin/$version"
```

Comment:
Maybe this should be the CDN URL instead? `https://upstreamartifacts.azureedge.net/$bpftrace_bin/$version`

Reply:
fixed and tested with this change